runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerBox.java

```diff
@@ -108,6 +109,15 @@
 		add(itemContainer, BorderLayout.CENTER);
 	}
 
+	private String buildToolTip(ItemManager itemManager, ItemStack item)
+	{
+		String name = itemManager.getItemComposition(item.getId()).getName();
+		int quantity = item.getQuantity();
+		long price = (long) itemManager.getItemPrice(item.getId()).getPrice() * item.getQuantity();
```

Comment:
Account also for coins here (like in calculatePrice)

Reply:
You should also account for `itemManager.getItemPrice` returning null. IIRC drops such as clue scrolls and noted items return null since they aren't tradeable.
I.E.
```
ItemPrice p = itemManager.getItemPrice(item.getId());
int price = item.getId() == ItemID.COINS_995 ? 1 : (p == null ? 0 : p.getPrice());
```